src/store/modules/AccountsModule.js

```diff
@@ -38,10 +43,31 @@ class AccountsModule extends VuexModule {
     return this.accounts.map(({ address }) => address);
   }
 
-  get isOtpMode() {
+  /**
+   * @deprecated
+   * @return {boolean}
+   */
+  get isAppOtp() {
     return !!this.settings.otpEnabled;
   }
 
+  get challengeType() {
```

Comment:
We have adapter. Do we need this logic? Can we return data from adapted settings?

Reply:
settings can be defined in localstorage, and it should work correct for old logged in users, that's why added this getter.

Reply:
I think, we should not handle settings stored locally, because it is not critical moment and in the any time we can redefine that without any tensions.

Reply:
i think not for this pr

Reply:
We should add guard for this case and drop local settings